Makefile

```diff
@@ -326,6 +328,12 @@ lint:
 lint-metrics:
 	./hack/prom_metric_linter.sh  --operator-name="kubevirt" --sub-operator-name="ssp"
 
+.PHONY: lint-monitoring
+lint-monitoring: $(MONITORING_LINTER)
+$(MONITORING_LINTER): $(LOCALBIN) ## Download monitoringlinter locally if necessary.
```

Comment:
You can skip this line, and write it as:
```
.PHONY: lint-monitoring
lint-monitoring: $(LOCALBIN)
```

The reason is, that if `$(MONITORING_LINTER)` file exists, `make` may not run this target. I think.

Reply:
Thanks, done. Now it's running also when it's already installed.